### PR TITLE
Added SiteMeta table

### PR DESCRIPTION
**Pull request by bruskiza**

### CODE REVIEW

**Comment by bruskiza** [comment]
@smn ready for review.

File: data.py [hunk 1]
```diff
@@ -38,6 +38,14 @@ class Meta:
         database = db
         db_table = 'nu_result'
 
+class NodeSiteMeta
```
Comment (smn) [inline data.py]: Needs to inherit `Model`

File: data.py [hunk 2]
```diff
@@ -38,6 +38,14 @@ class Meta:
...
+    node_id = IntergerField(index=True)
+    ip_address = CharField(null=True. max_length=40)
```
Comment (smn) [inline data.py]: typo after `True`

File: data.py [hunk 3]
```diff
@@ -38,6 +38,14 @@ class Meta:
         database = db
         db_table = 'nu_result'
 
+class NodeSiteMeta
+    node_id = IntergerField(index=True)
```
Comment (smn) [inline data.py]: typo in `Integer`

File: data.py [hunk 4]
```diff
@@ -60,6 +68,10 @@ def create_tables():
         print "Created nu_result table."
         db.create_tables([Result])
 
+    if not Result.table_exists():
```
Comment (smn) [inline data.py]: Checking the wrong table here, needs to be `NodeSiteMeta`, not `Result`.

**Comment by smn** [comment]
:+1:

**Pull request merged**